monkey/monkey_island/cc/ui/src/components/ui-components/Error404Modal.js

```diff
@@ -0,0 +1,45 @@
+import {Modal} from 'react-bootstrap';
+import React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faExclamationTriangle, faTimes} from '@fortawesome/free-solid-svg-icons';
+import {Button, Row, Col} from 'react-bootstrap';
+
+
+class Error404Modal extends React.PureComponent {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showModal: this.props.showModal,
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props !== prevProps) {
+      this.setState({
+        showModal: this.props.showModal,
+      })
+    }
+  }
+
+  render = () => {
+    return (
+      <Modal show={this.state.showModal} onHide={() => this.props.onClose()}>
+        <Modal.Header closeButton>
+          <Modal.Title> Uh oh... </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <div style={{'marginTop': '1em', 'marginBottom': '1em'}}>
+            <p className="alert alert-danger">
+              <FontAwesomeIcon icon={faExclamationTriangle} style={{'marginRight': '5px'}}/>
+              The server returned a 404 (NOT FOUND) response.
```

Comment:
Can we add a custom message to this component and just rename it ErrorModal? We can probably reuse it in other place too.